### PR TITLE
IBM/Blade Networks/Lenovo Support

### PR DESCRIPTION
**Pull request by jackson-tim**

### CODE REVIEW

File: lib/oxidized/model/ibm.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    cfg.gsub! /^(\s+(?:password|secret)) (?:\d )?\S+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(tacacs-server \S+host \S+ ekey) (\S+)/, '\\1 <secret hidden>'
+    cfg
+  end
+
+  cmd 'show version' do |cfg|
+    comments = []
+    comments << cfg.lines.first
+    lines = cfg.lines
+    lines.each_with_index do |line,i|
```
Comment (danilopopeye) [inline lib/oxidized/model/ibm.rb]: why not use `#each`? the index doesn't look like is needed here

File: lib/oxidized/model/ibm.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        if line.match /^Software Version\s+(.+)\s+\(FLASH (.+)\).*/
+            comments << "Software: #{$1}, Booted from: #{$2}"
+        end
+
+        if line.match /^Boot kernel version (.+)/
```
Comment (danilopopeye) [inline lib/oxidized/model/ibm.rb]: you should join all these `if` so we don't need to test three times each line 😉

Comment (danilopopeye) [inline lib/oxidized/model/ibm.rb]: now that I come to think about it, why not just comment the whole command output?

**Comment by laf** [comment]
@jackson-tim Any feedback on the requested changes for this?

**Comment by zombah** [comment]
This looks like same as in #867 
Regarding IBM site correct name for OS is BladeOS

**Comment by mortzu** [comment]
Can you please fix the conflict and add your change to CHANGELOG

**Pull request closed without merging**